src/Compilers/Core/Portable/Operations/ControlFlowGraphExtensions.cs

```diff
@@ -43,7 +42,7 @@ public static ControlFlowGraph GetLocalFunctionControlFlowGraphInScope(this Cont
         /// Gets or creates a control flow graph for the given <paramref name="anonymousFunction"/> defined in
         /// the given <paramref name="controlFlowGraph"/> or any of it's parent control flow graphs.
         /// </summary>
-        public static ControlFlowGraph GetAnonymousFunctionControlFlowGraphInScope(this ControlFlowGraph controlFlowGraph, IFlowAnonymousFunctionOperation anonymousFunction, CancellationToken cancellationToken = default)
+        public static ControlFlowGraph GetAnonymousFunctionControlFlowGraphInScope([DisallowNull] this ControlFlowGraph? controlFlowGraph, IFlowAnonymousFunctionOperation anonymousFunction, CancellationToken cancellationToken = default)
```

Comment:
📝 Outcome of this public API would be the same as https://github.com/dotnet/roslyn/pull/49271/files#r521113616